library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -118,24 +132,42 @@ private int getDetailedState() {
     return state;
   }
 
+  private void removeNamedPlan(NamedPlan plan, String name, Performer.NamedPlanPerformance performer) {
+    if (performer != null) {
+      performer.removePlan(plan, name);
```

Comment:
:facepalm: I should have caught this, but went on autopilot and just was using the `addNamedPlan` as a template for this interface.